src/linux_service.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Set of utilities related to managing Linux services."""
+
+import logging
+import os
+from subprocess import CalledProcessError
+from typing import Optional
+
+from jinja2 import Template
+
+from utils import shell
+
+SERVICE_TEMPLATE = "./templates/service.j2"
+
+logger = logging.getLogger(__name__)
+
+
+class Service:
+    """Class for representing a Linux Service."""
+
+    def __init__(self, name: str):
+        """Sets service name."""
+        self.name = name
+
+    def is_active(self) -> bool:
+        """Returns whether service is active."""
+        try:
+            response = shell(f"systemctl is-active {self.name}")
```

Comment:
You have a `_systemctl` function that could be used for that if it returned.

Reply:
Good point, fixed